soil/templates/soil/partials/dl_status.html

```diff
@@ -1,4 +1,5 @@
 {% load i18n %}
+{% load hq_shared_tags %}
```

Comment:
if we are consciously breaking that abstraction and not subtree merging then I think we should update the readme saying that this is basically part of commcare hq now